src/pages/profile/index.js

```diff
@@ -0,0 +1,66 @@
+import { useEffect, useRef } from "react";
+import { useHistory } from "react-router";
+import { SubmitButton } from "../../components/elements";
+import { AUTH } from "../../constants";
+import { getUserData, updateUserData } from "../../services/userService";
+import "./profile.css";
+
+export function Profile() {
+  const history = useHistory();
+  const nameRef = useRef();
+  const emailRef = useRef();
+  const formRef = useRef();
+  useEffect(() => {
+    getUserData()
+      .then((userData) => {
+        nameRef.current.value = userData.name;
+        emailRef.current.value = userData.email;
+      })
+      .catch((errorMessage) => {
+        if (errorMessage === "You are not logged in") history.replace(AUTH);
+      });
+  });
+  const onSubmitHandler = (event) => {
+    event.preventDefault();
+    const userData = new FormData(formRef.current);
+    if (userData.get("password") === "") userData.delete("password");
+    updateUserData(userData)
+      .then((res) => alert(res.message))
+      .catch((exception) => alert(exception));
+  };
+  return (
+    <div className="profile-box">
+      <h1 className="profile-title">Profile</h1>
+      <form className="profile-form" ref={formRef} onSubmit={onSubmitHandler}>
+        <div className="profile-form-row">
+          <h3 className="profile-form-label">Name: </h3>
```

Comment:
I can't remember how we already have done it, but we should use the `label` element with the `for` property to display labels of inputs. It helps browsers to identify respective input labels and when we click on label it focuses on respective input. This also helps screen readers for visually disabled persons.